src/tls13.c

```diff
@@ -3672,7 +3672,7 @@ int SendTls13ClientHello(WOLFSSL* ssl)
 
     ssl->options.clientState = CLIENT_HELLO_COMPLETE;
 
-#ifdef WOLFSSL_CALLBACKS
+#if defined(WOLFSSL_CALLBACKS) || defined(OPENSSL_EXTRA)
```

Comment:
This looks good! that way the behavior should be the same with and without `WOLFSSL_CALLBACKS` if `OPENSSL_EXTRA` is set.